app/src/main/java/org/mozilla/reference/browser/database/Topsite.kt

```diff
@@ -0,0 +1,21 @@
+package org.mozilla.reference.browser.database
+
+/**
+ * This class catches basic informations about a "topsite"
+ *
+ * @author Khaled Tantawy
+ * @author Stefano Pacifici
+ */
+class Topsite
+/**
+ *
+ * @param id the urls database id for this topsite
+ * @param url the topsite url
+ * @param domain the pre-extracted domain (from url)
+ * @param title the title of the page
+ */(
+     val id: Long,
+     val url: String,
+     val domain: String,
+     val title: String?
+ )
```

Comment:
Please add license header, remove authors. Are you sure this is the right way to indent the documentation.